utils/v2/connector/impl/RedStonePriceFeed.ts

```diff
@@ -1,5 +1,6 @@
 import { PriceFeedConnector, SignedPriceDataType } from "../PriceFeedConnector";
 import axios from "axios";
+import bluebird from "bluebird";
```

Comment:
Why are you using bluebird, is it better than Promise from ES2021 (https://javascript.plainenglish.io/promise-any-and-promise-race-in-es2021-3250733b48eb) or is it not fully supported by browsers?

Reply:
Bluebird is more efficient and has features that are not available in standard JS/TS. E.g. promise cancellation, that can be added in future to redstone-evm-connector. I also had some traoubles with running ES2021 on my machine, so very probably it can case similar problems with compactibility on other devices